### PR TITLE
update(docs): update changelog to Universum 0.19.1

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,30 @@
 Change log
 ==========
 
+0.19.1 (2020-11-06)
+-------------------
+
+New features
+~~~~~~~~~~~~
+
+* **config:** add classes :class:`~universum.configuration_support.Step` and
```
Comment (i-keliukh) [inline CHANGELOG.rst:10]: add :class:`~universum.configuration_support.Step` and :class:`~universum.configuration_support.Configuration` classes

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+  :class:`~universum.configuration_support.Configuration` to :doc:`configuration support <configuration_support>`
+  for type checking and increased usability
+* **config:** default config name ``.universum.py`` is :doc:`now supported <configuring>` along with
```
Comment (i-keliukh) [inline CHANGELOG.rst:13]: I would say --config is now not mandatory. If it is not specified, the default value is used.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,6 +1,30 @@
...
+  ``-cfg``/``--config`` :doc:`command line argument <args>`
+* **config:** add ``universum init`` :doc:`command <additional_commands>` for automatic configuration file creation
+* **add ``universum run`` :doc:`command <additional_commands>` to launch Universum in Non-CI mode**
+* **add printing multiple command line argument errors**
```
Comment (i-keliukh) [inline CHANGELOG.rst:17]: More details are needed for this item

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,6 +1,30 @@
...
+
+Bug fixes
+~~~~~~~~~
+
+* **swarm:** fix detecting latest review version
+* **out:** do not print Unicode characters in non-Unicode locales
+* **jenkins_plugin:** return missing 'Failed' line with timestamps plugin
+* **use launched Python version instead of hardcoded**
```
Comment (i-keliukh) [inline CHANGELOG.rst:25]: Where we use lanched Python version?

**Comment by k-dovgan** [comment]
https://universum.readthedocs.io/en/update-changelog/changelog_ref.html

File: CHANGELOG.rst [hunk 5]
```diff
@@ -1,6 +1,29 @@
...
+
+New features
+~~~~~~~~~~~~
+
+* **config:** add :class:`~universum.configuration_support.Step` and :class:`~universum.configuration_support.Configuration`
+  classes to :doc:`configuration support <configuration_support>` for type checking and increased usability
+* **config:** ``-cfg``/``--config`` :doc:`command line argument <args>` is no longer mandatory; if not specified,
+  the default value (``.univerum.py`` in project root) is used
+* **config:** add ``universum init`` :doc:`command <additional_commands>` for automatic configuration file creation
+* **add ``universum run`` :doc:`command <additional_commands>` to launch Universum in Non-CI mode**
```
Comment (i-keliukh) [inline CHANGELOG.rst:15]: It looks like entire string made bold messes up the sphinx rendering, see https://universum.readthedocs.io/en/update-changelog/changelog_ref.html

File: CHANGELOG.rst [hunk 6]
```diff
@@ -1,6 +1,29 @@
...
+* **if more than one command line argument caused an error, all of error messages are now shown** (not only the first one)
```
Comment (i-keliukh) [inline CHANGELOG.rst:16]: Please make less amount of text bold and also remove parentheses

**Review by i-keliukh — APPROVED** [review]

**Pull request merged**